appendix-basic65.tex

```diff
@@ -3658,18 +3661,18 @@ \subsection{ERR\$}
 
                   This function is typically used in a {\bf TRAP} routine, where the error number is taken from the reserved variable {\bf ER}.
 
-\item [Remarks:]  A BASIC error number out of range (1 -- 44) will produce an \screentext{OK}.
+\item [Remarks:]  A BASIC error number out of range (1 -- 44) will produce an \screentext{OK}
```

Comment:
As above, we do need sentence ending punctuation when it's a sentence. The period is probably fine in this case because "OK" is not typed by the user.

To clarify what I mean, I consider a colon and some typing text to be sentence ending punctuation, and therefore a way to avoid putting a period after typing text. The typing text itself is not sentence ending, but it can hang after the colon. Like so:

```
To start the program, type the following command: \texttt{RUN} Your program runs until it is stopped.
```